libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_common.h

```diff
@@ -58,12 +58,20 @@
 
 /* Enable/Disable test cases */
 /* AFQP test */
-#define ENABLE_TC_WRITE_LONG              ( 1 )
-#define ENABLE_TC_ADD_INCLUDED_SERVICE    ( 0 )
-#define ENABLE_TC_SECONDARY_SERVICE       ( 0 )
+#define ENABLE_TC_WRITE_LONG                                      ( 1 )
+#define ENABLE_TC_ADD_INCLUDED_SERVICE                            ( 0 )
+#define ENABLE_TC_SECONDARY_SERVICE                               ( 0 )
 /* Integration test */
-#define ENABLE_TC_CALLBACK_NULL_CHECK     ( 0 )
-
+#define ENABLE_TC_CALLBACK_NULL_CHECK                             ( 1 )
+#define ENABLE_TC_ADD_CHARACTERISTIC_IN_CALLBACK                  ( 0 )
+#define ENABLE_TC_ADVERTISE_WITHOUT_PROPERTISE                    ( 1 )
+#define ENABLE_TC_ADVERTISE_WITH_16BIT_SERVICEUUID                ( 1 )
+#define ENABLE_TC_ADVERTISE_WITH_MANUFACTUREDATA                  ( 1 )
+#define ENABLE_TC_ADVERTISE_INTERVAL_CONSISTENT_AFTER_BT_RESET    ( 1 )
+#define ENABLE_TC_WRITE_NOTIFICATION_SIZE_GREATER_THAN_MTU_3      ( 1 )
+#define ENABLE_TC_INTEGRATION_CONNECTION_TIMEOUT                  ( 1 )
```

Comment:
Some test cases has prefix "INTEGRATION". Can we prefix the TC with category for all test cases to be consistent.

Reply:
sure